.github/workflows/zcash-lightwalletd.yml

```diff
@@ -36,7 +53,11 @@ jobs:
       - uses: actions/checkout@v3.0.2
         with:
           repository: adityapk00/lightwalletd
-          ref: 'master'
+          # Temporarily use an earlier lightwalletd version,
+          # to check if commit e146dbf5c2860e535fdbe2ba5ab20c4744d7c941 contains a mempool refresh deadlock bug.
+          #
+          # TODO: switch back to 'master' after the bug is resolved
+          ref: 'c1bab818a683e4de69cd952317000f9bb2932274'
```

Comment:
@gustavovalverde is this staying in?

Reply:
To be honest, is working without it, as the Dockerfile is building with the image available from main. But the test sometimes hangs, and might be because of this; but I'm not the one to make the call so I'll invoke @teor2345

Reply:
I'd like to leave this change in, and see if it improves the hangs.

If it does, we'll have a chat to Aditya about a permanent fix.